backend/store.py

```diff
@@ -68,3 +68,21 @@ def get_random(cls) -> Path:
         files = cls.get_logs()
 
         return Path(random.choice(files))
+
+    @classmethod
+    def get_stats(cls) -> dict:
+        """Retrieve basic statistics about submitted reports.
+        """
+        files = cls.get_logs()
+        stats = {
+            "total_reports" : len(files),
+            "avg_snippets" : 0.0
+        }
+        for path in files:
+            with open(path, encoding='utf-8') as f:
+                report = json.load(f)
```

Comment:
I am getting `{"error":"Server error: 500","description":"Expecting value: line 1 column 1 (char 0)"}` from /stats

traceback:

```
...
  File "/usr/lib/python3.11/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/lib/python3.11/site-packages/starlette/routing.py", line 66, in app
    response = await func(request)
               ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/fastapi/routing.py", line 241, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/fastapi/routing.py", line 169, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/anyio/to_thread.py", line 28, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(func, *args, cancellable=cancellable,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/anyio/_backends/_asyncio.py", line 818, in run_sync_in_worker_thread
    return await future
           ^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/anyio/_backends/_asyncio.py", line 754, in run
    result = context.run(func, *args)
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/log-detective-website/backend/api.py", line 318, in get_report_stats
    return Storator3000.get_stats()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/log-detective-website/backend/store.py", line 83, in get_stats
    report = json.load(f)
             ^^^^^^^^^^^^
  File "/usr/lib64/python3.11/json/__init__.py", line 293, in load
    return loads(fp.read(),
           ^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
```

Reply:
ok that was problem in my data, sorry for the noise